lib/screens/information_screen.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:kledd/lang/my_localizations.dart';
+import '../widgets/information_segment.dart';
```

Comment:
Reorder imports. Should be ordered dart related, packages and then local files separated by a blank line.